data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "origin": {
+    "date_human-readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375",
+    "language": "Greek",
+    "place": {}
+  },
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "authors": [
+    {
+      "id": "https://sinaimanuscripts.library.ucla.edu/person/55",
+      "role": "author"
+    }
+  ],
+  "associated_persons": [],
+  "authority_files": [
+    {
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
+  "witnesses": ["ark:/21198/dfdi53234x"],
+  "references": [],
+  "contributor": ["gkessel"]
+}
```

Comment:
Other fields to include?:

- need field relating works? (dcterms:isPartOf for part-to-whole like the Gospels and a Gospel; 'comments_on' for relating Hexaemeron to Genesis, etc.)
- incipits, explicits, etc. from a standard edition (or an authority file like Syriaca?)? This gets tricky and probably unnecessary as these are more to provide authority control for the witnesses we come across in Sinai mss. (interested users could pull the incipits together from the witnesses/textual_artifacts)